pages/portfolio.html

```diff
@@ -4,9 +4,63 @@
   <meta charset="UTF-8">
   <meta http-equiv="X-UA-Compatible" content="IE=edge">
   <meta name="viewport" content="width=device-width, initial-scale=1.0">
-  <title>Document</title>
+  <title>Portfolio | Uyen Vong</title>
+  <link rel="stylesheet" href="../styles/style.css">
 </head>
 <body>
-  
+  <header>
+    <h1>projects</h1>
+  </header>
+  <nav>
+    <ul>
+      <li>
+        <a href="/pages/index.html">Home</a>
+      </li>
+      <li>
+        <a href="/pages/about.html">About</a>
+      </li>
+      <li>
+        <a href="/pages/portfolio.html">Projects</a>
+      </li>
+    </ul>
+  </nav>
+  <main>
+    <article>
+      <h2>Here are some projects that I've worked on</h2>  
+      <ul>
+        <li>
+          <h3><a href="https://github.com/uvong/viewing-party">Viewing Party</a></h3>
+          <ul>
+            <li>Worked with nested loops</li>
+            <li>Worked with nested data structures</li>
+          </ul>
+        </li>
+        <li>
+          <h3><a href="https://github.com/uvong/swap-meet">Swap Meet</a></h3>
+          <ul>
+            <li>Implemented Python OOP</li>
+            <li>Implemented inheritance</li>
+          </ul>
+        </li>
+        <li>
+          <h3><a href="https://github.com/uvong/task-list-api">Task List</a></h3>
+          <ul>
+            <li>Implemented Slack's API</li>
+            <li>Created and implemented models</li>
+            <li>Created RESTful API</li>
+          </ul>
+        </li>
+      </ul>
+    </article>
+  </main>
+  <footer>
```

Comment:
Nice semantic tags! Once we study react, we can create just one "footer" component and import it across several files, which helps with DRY. Just a preview!